pkg/reconciler/kubernetes/tektonconfig/controller.go

```diff
@@ -88,3 +93,13 @@ func NewExtendedController(generator common.ExtensionGenerator) injection.Contro
 		return impl
 	}
 }
+
+// customEnq adds `config` in work queue so that whenever a namespace
+// event occurs, the reconciler get triggered. The reconciler will only
+// get triggered only when a TektonConfig instance with name `config`
+// already exist.
```

Comment:
mention that this is required because we want to get our TektonConfig reconciler triggerd for already existing and new namespaces, without manualy intervention like `adding a label/annotion` on namespace to make it manageable by Tekton controllers. 

in addition could you take care of:
- `s/adds 'config' in work queue/adds an event with name `config` in work queue/`
- `s/the reconciler get triggered./the TektonConfig reconciler get triggered./`
- remove 
`The reconciler will only// get triggered only when a TektonConfig instance with name `config`// already exist.` because the reconciler will be triggered for TektonConfig CR instances with any name. Inside the reconciler we just have a check to skip reconcile for all instances of TektonConfig with anyother name other than `config`.